<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TexasWidget</class>
 <widget class="QWidget" name="TexasWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1170</width>
    <height>698</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TexasWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>A</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" rowspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labelImg1">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>320</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>320</height>
        </size>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelImg2">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>320</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>320</height>
        </size>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelImg3">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>320</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>320</height>
        </size>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelImg4">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>320</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>320</height>
        </size>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelImg5">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>320</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>320</height>
        </size>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="btn_Gen">
     <property name="text">
      <string>生成</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="lineEditBrand">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>269</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>B</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" rowspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="labelImg11">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>320</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>320</height>
        </size>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelImg12">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>320</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>320</height>
        </size>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelImg13">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>320</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>320</height>
        </size>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelImg14">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>320</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>320</height>
        </size>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelImg15">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>320</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>320</height>
        </size>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="labelResult">
     <property name="text">
      <string>结果</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLineEdit" name="lineEditBrand2">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>269</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
